Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -596,7 +596,14 @@ end
 				HasPluginChannel = { Params = "ChannelName", Return = "bool", Notes = "Returns true if the client has registered to receive messages on the specified plugin channel." },
 				IsUUIDOnline = { Params = "UUID", Return = "bool", IsStatic = true, Notes = "Returns true if the UUID is generated by online auth, false if it is an offline-generated UUID. We use Version-3 UUIDs for offline UUIDs, online UUIDs are Version-4, thus we can tell them apart. Accepts both 32-char and 36-char UUIDs (with and without dashes). If the string given is not a valid UUID, returns false."},
 				Kick = { Params = "Reason", Return = "", Notes = "Kicks the user with the specified reason" },
+				SendHideTitle = { Params = "", Return = "", Notes = "Sends to hide the title." },
 				SendPluginMessage = { Params = "Channel, Message", Return = "", Notes = "Sends the plugin message on the specified channel." },
+				SendResetTitle = { Params = "", Return = "", Notes = "Sends to reset the title." },
+				SendSetSubTitle = { Params = "SubTitle", Return = "", Notes = "Send the subtitle but does not show it. Use SendTitleTimes to show the title." },
+				SendSetRawSubTitle = { Params = "RawSubTitle", Return = "", Notes = "Sends the raw subtitle but does not show it. Use SendTitleTimes to show the title." },
+				SendSetTitle = { Params = "Title", Return = "", Notes = "Sends the title but does not show it. Use SendTitleTimes to show the title." },
+				SendSetRawTitle = { Params = "RawTitle", Return = "", Notes = "Sends the raw title but does not show it. Use SendTitleTimes to show the title." },
+				SendTitleTimes = { Params = "FadeInTicks, DisplayTicks, FadeOutTicks", Return = "", Notes = "Sends the title times and shows the title." },
```

Comment:
`Sends the request to display the Title and Subtitle, previously set with SendSetTitle, SendSetRawTitle, SendSetSubTitle and SendSetRawSubTitle, to the client.`